package.js

```diff
@@ -41,11 +41,11 @@ Package.onTest(function (api) {
   // Add code coverage
   api.use([
     'lmieulet:meteor-legacy-coverage',
-    'lmieulet:meteor-coverage@4.1.0',
-    'meteortesting:mocha@2.1.0'
+    'lmieulet:meteor-coverage@4.1.0 || 5.0.0',
+    'meteortesting:mocha@2.1.0 || 3.0.0'
```

Comment:
Not sure if we should explicitly use `meteortesting:mocha@3.2.0` since before this version there was a little mess with the versions

Reply:
Improving!